libs/ballot-interpreter-vx/src/utils/qrcode.ts

```diff
@@ -1,14 +1,33 @@
 import { detect as detectMetadata } from '@votingworks/ballot-encoder';
 import { Rect, Size } from '@votingworks/types';
-import { detect as qrdetect } from '@votingworks/qrdetect';
 import makeDebug from 'debug';
 import jsQr from 'jsqr';
-import { decode as quircDecode, QRCode } from 'node-quirc';
+import { QRCode } from 'node-quirc';
 import { DetectQrCodeResult } from '../types';
 import { crop } from './crop';
 
 const debug = makeDebug('ballot-interpreter-vx:qrcode');
 
+// Loads these libraries dynamically because they are not available in
+// the browser.
```

Comment:
Can you say a bit more about what this means? I don't totally get what "not available in the browser" means

Reply:
I'll update the comment. It means that these modules have NodeJS native extensions that cannot be loaded into the browser.